### ISSUE TITLE
Allow Github logins to select between emails

### ISSUE THREAD

**Issue by torgeirl** [vfeskov/gitpunch#36]
Regular users can control which email they receive which notifications to by registering with multiple emails (essential having more than one account):

> - Register with multiple emails (e.g., email+important@example.com) to have **more than one list**

Users logged in with Github seems to be constrained to their main email, and then having to register another account with regular login if they want more control over where the notifications are sent.

Doesn't the Github OAuth API allow you to request other email addresses on that account? I would be great if I could control everything at one place (single account), and for each repo just could toggle between my emails for where the notification would be sent.

**Comment by vfeskov** [comment]
@torgeirl yes you're right, github does allow me to request all emails, and it'd be natural to see your github emails under one account on gitpunch, Im definitely going to do it

**Comment by vfeskov** [comment]
i guess i was too hasty closing this issue, i assumed it was similar to this https://github.com/vfeskov/gitpunch/issues/63 (changing email on github lead to terrible things)
